ui/pages/confirmation/components/confirmation-network-switch/confirmation-network-switch.js

```diff
@@ -20,82 +18,54 @@ import {
 } from '../../../../../shared/constants/network';
 import { getProviderConfig } from '../../../../ducks/metamask/metamask';
 
-export default function ConfirmationNetworkSwitch({ newNetwork }) {
-  const { chainId, nickname, type } = useSelector(getProviderConfig);
-
+const NetworkIcon = ({ chainId, nickname }) => {
   return (
-    <Box
-      className="confirmation-network-switch"
-      display={Display.Flex}
-      height={BlockSize.Full}
-      justifyContent={JustifyContent.center}
-      marginTop={8}
-    >
-      <Box
-        className="confirmation-network-switch__icon"
-        display={Display.Block}
-      >
+    <Box className="confirmation-network-switch__icon-wrapper">
+      <Box className="confirmation-network-switch__icon">
         {chainId in CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP ? (
-          <SiteIcon
-            icon={CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP[chainId]}
+          <AvatarFavicon
+            margin="auto"
+            src={CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP[chainId]}
+            size={AvatarFaviconSize.Xl}
             name={nickname}
-            size={64}
           />
         ) : (
-          <div className="confirmation-network-switch__unknown-icon">
-            <i className="fa fa-question fa-2x" />
-          </div>
+          <i className="fa fa-question fa-2x" />
         )}
-        <Typography
-          color={TextColor.textDefault}
-          variant={TypographyVariant.H6}
-          fontWeight={FontWeight.Normal}
-          align={TextAlign.Center}
-          boxProps={{
-            display: Display.Flex,
-            justifyContent: JustifyContent.center,
-          }}
-        >
-          {nickname || NETWORK_TO_NAME_MAP[type]}
-        </Typography>
       </Box>
-      <Box
-        className="confirmation-network-switch__center-icon"
-        display={Display.Flex}
-        alignItems={AlignItems.center}
-        justifyContent={JustifyContent.center}
+      <Text
+        className="confirmation-network-switch__text"
+        color={TextColor.textDefault}
+        variant={TypographyVariant.H6}
       >
+        {nickname}
+      </Text>
+    </Box>
+  );
+};
+
+NetworkIcon.propTypes = {
+  chainId: PropTypes.string.isRequired,
+  nickname: PropTypes.string.isRequired,
+};
+
+export default function ConfirmationNetworkSwitch({ newNetwork }) {
+  const { chainId, nickname, type } = useSelector(getProviderConfig);
+
+  return (
+    <Box className="confirmation-network-switch" height={BlockSize.Full}>
```

Comment:
Instead of the attached CSS here I would suggest using the style utility props
```
  display: flex;
  justify-content: center;
  margin-top: 8px;
```
can be written using props and design system enums
```
import {
  JustifyContent,
  Display,
  FlexDirection,
} from '../../../../helpers/constants/design-system';
```

```suggestion
    <Box className="confirmation-network-switch" height={BlockSize.Full} display={Display.Flex} justifyContent={JustifyContent.center} marginTop={2} >
```